test/forked/rails_rake_full_stack_test.rb

```diff
@@ -0,0 +1,21 @@
+require File.expand_path('../rails_test_helper', File.dirname(__FILE__))
+
+class RailsRakeFullStackTest < Minitest::Test
+
+  test 'rake tasks shows coverage properly within eager_loading' do
+    system("COVERBAND_CONFIG=./test/rails5_dummy/config/coverband.rb bundle exec rake -f test/rails5_dummy/Rakefile middleware")
```

Comment:
Shelling out allows us to test rails initialization from a rake task.

Reply:
This is awesome, I think we can add a few more like this (not in this PR) to cover all the weird load order bugs we have been sorting out.

Reply:
Agreed. Whenever we think of tests we are missing, let's add them to issues